src/index.d.ts

```diff
@@ -4,6 +4,15 @@ interface SnippetOptions {
   debug?: boolean;
   host?: string;
   script?: string;
+  iFrameOption?: IFrameOption;
+}
+
+// recording options for iframes
+// `RecordCrossDomainIFrame` has security implications
+// see https://help.fullstory.com/hc/en-us/articles/360020622514
+export enum IFrameOption {
+  RecordCrossDomainIFrame = '_fs_run_in_iframe',
+  RecordOnlyIFrame = '_fs_is_outer_script',
```

Comment:
A couple of thoughts:

- `RecordOnlyIFrame` - still feels like this could be more descriptive. How do you feel about `RecordOnltHostiFrame` or `RecordOnltThisiFrame`?
- I originally included a default enum value of "record same domain iFrame". What are your thoughts on adding this back? It's basically a no-op but completely describes the recording options for iFrames. An Enum like this perhaps:

```
export enum IFrameOption {
  RecrodSameDomainIFrame = '_fs_iframe_default_no_op',
  RecordCrossDomainIFrame = '_fs_run_in_iframe',
  RecordOnlyIFrame = '_fs_is_outer_script',
}
```

Reply:
We should refine `RecordOnlyIFrame`. The term "host" still throws me off as I'm not clear what that pertains to. `RecordOnlyThisIFrame` almost implies I can control which iFrames to record, but I am liking this option the most (and it seems to cover option 4 in the KB article).  Also, case of `I` or `i` in the var should be determined.

I think a pro for having `RecordSameDomainIFrame` is that is clarifies the default behavior.  The cons however are: a) there's no way to turn off this feature (since FullStory records same domain iFrames by default, right) b) the options were first designed as mutually exclusive so if you set `RecordCrossDomainIFrame`, a dev might wonder, what's the state of `RecordSameDomainIFrame`?  To fix that, `iFrameOption` becomes `iFrameOptions` to accommodate `RecordCrossDomainIFrame` and `RecordCrossDomainIFrame`.  I'm now thinking we should validate the mutual exclusivity of these entirely.

Reply:
I see the quandary that the `RecordSameDomainIFrame` option presents as well, but the problem persists if it's not presented as an option too. The question a dev might always ask is "will setting `RecordCrossDomainIFrame` flip off `RecordSameDomainIFrame`?

`RecordOnlyIFrame` doesn't specify which iFrame. A dev could reasonably interpret this as "record only the iframes on my site"

Let's F2F

Reply:
While we're at it, we should pluralize these too: `RecordSameDomainIFrames` and `RecordCrossDomainIFrames`.